src/Tags/Concerns/GetsQueryResults.php

```diff
@@ -38,7 +39,9 @@ protected function paginatedResults($query, $perPage)
             $this->queryPaginationFriendlyOffset($query, $offset);
         }
 
-        $paginator = $query->paginate($perPage);
+        $columns = $query instanceof EloquentQueryBuilder ? [] : ['*'];
```

Comment:
Did `['*']` not work when using files? I would think it's fine as it's the default anyway. https://github.com/statamic/cms/blob/5e3de537f7e7821b145cfae547be29cfc6e985e4/src/Query/Builder.php#L524

Reply:
Yeah I wasn't sure to be honest. `['*']` works perfectly normally, but it's not the default in the eloquent query builder so I wasn't sure if it needed to accommodate that?

https://github.com/statamic/cms/blob/5e3de537f7e7821b145cfae547be29cfc6e985e4/src/Query/EloquentQueryBuilder.php#L72

Reply:
Oh sorry I was reading that backwards. 😵 I thought you were passing the asterisk when it's eloquent.

I think the default value of `[]` in the eloquent query builder is wrong, it should be `[*]`. We can tackle that separately. 

For this PR you can get rid of the condition and just pass along `[*]`.

Reply:
Ah OK, reverted!